packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -80,10 +85,12 @@ library TaikoData {
         bytes signature;
     }
 
+    struct BlockParams {
```

Comment:
Going forward we can put more parameter inside this struct without changing the proposeBlock's ABI